lib/cocoapods/generator/embed_frameworks_script.rb

```diff
@@ -76,10 +76,12 @@ def script
             local swift_runtime_libs
             swift_runtime_libs=$(xcrun otool -LX "${CONFIGURATION_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}/$1/${basename}" | grep @rpath/libswift | sed -E s/@rpath\\\\/\\(.+dylib\\).*/\\\\1/g | uniq -u  && exit ${PIPESTATUS[0]})
             for lib in $swift_runtime_libs; do
-              echo "rsync -av \\"${SWIFT_STDLIB_PATH}/${lib}\\" \\"${destination}\\""
-              rsync -av "${SWIFT_STDLIB_PATH}/${lib}" "${destination}"
-              if [ "${CODE_SIGNING_REQUIRED}" == "YES" ]; then
-                code_sign "${destination}/${lib}"
+              if [ -n "${destination}" ]; then
```

Comment:
Because I'm a terrible shell programmer? I thought that checked if there was a file there

-Samuel E. Giddins

On Feb 17, 2015, at 8:24 AM, Kyle Fuller <notifications@github.com> wrote:

In lib/cocoapods/generator/embed_frameworks_script.rb:

> @@ -76,10 +76,12 @@ def script
>              local swift_runtime_libs
>              swift_runtime_libs=$(xcrun otool -LX "${CONFIGURATION_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}/$1/${basename}" | grep @rpath/libswift | sed -E s/@rpath\\\\/\\(.+dylib\\).*/\\\\1/g | uniq -u  && exit ${PIPESTATUS[0]})
>              for lib in $swift_runtime_libs; do
> -              echo "rsync -av \\"${SWIFT_STDLIB_PATH}/${lib}\\" \\"${destination}\\""
> -              rsync -av "${SWIFT_STDLIB_PATH}/${lib}" "${destination}"
> -              if [ "${CODE_SIGNING_REQUIRED}" == "YES" ]; then
> -                code_sign "${destination}/${lib}"
> +              if [ -n "${destination}" ]; then
So is there any particular reason why you're checking if the string is not empty?

—
Reply to this email directly or view it on GitHub.